docs/using-the-nodejs-wrapper/UsingTheNodejsWrapper.md

```diff
@@ -50,7 +50,9 @@ These parameters are applicable to any instance of the AWS Advanced NodeJS Wrapp
-| `mysqlQueryTimeout`            | `number`  | No                                                                               | Query timeout in milliseconds. This is only applicable when using the AwsMySQLClient. To set query timeout for the AwsPGClient, please use the built-in `query_timeout` parameter. See the `node-postgres` [documentation](https://node-postgres.com/apis/client) for more details..                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          | 20000                                                                                                                                                                        |
+| ~~`mysqlQueryTimeout`~~        | `number`  | No                                                                               | This parameter has been deprecated, applications should use the `wrapperQueryTimeout` parameter instead. <br/><br/> Query timeout in milliseconds. This is only applicable when using the AwsMySQLClient. To set query timeout for the AwsPGClient, please use the built-in `query_timeout` parameter. See the `node-postgres` [documentation](https://node-postgres.com/apis/client) for more details.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       | 20000                                                                                                                                                                        |
```

Comment:
Need to mention the version. e.g. `has been deprecated since version....`
Can we add a `version` column so users don't try to use newer parameters on older driver versions